### PR TITLE
Update prerequisites.adoc

### PR DESCRIPTION
**Pull request by hodyroff**
Added IDP, Authentication/Authorisation.

### CODE REVIEW

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 1]
```diff
@@ -240,6 +240,11 @@ For _best performance_, _stability_, _support_, and _full functionality_ we offi
 * openSUSE Leap 15.2 and 15.3
 * Ubuntu 20.04 and 22.04
 
+== Authorization and Authorisation (IDP)
+ownCloud Infinite Scale is dependent on Open ID Connect, therefor a connection to an idendity provider (IDP) is mandatory.
```
Comment (phil-davis) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:244]:
```suggestion
ownCloud Infinite Scale is dependent on Open ID Connect, therefore a connection to an identity provider (IDP) is mandatory.
```

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 2]
```diff
@@ -240,6 +240,11 @@ For _best performance_, _stability_, _support_, and _full functionality_ we offi
 * openSUSE Leap 15.2 and 15.3
 * Ubuntu 20.04 and 22.04
 
+== Authorization and Authorisation (IDP)
```
Comment (phil-davis) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:243]:
```suggestion
== Authorization and Authentication (IDP)
```

Maybe Authentication is meant?

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 3]
```diff
@@ -240,6 +240,11 @@ For _best performance_, _stability_, _support_, and _full functionality_ we offi
...
+ownCloud Infinite Scale is dependent on Open ID Connect, therefor a connection to an idendity provider (IDP) is mandatory.
+Konnect is included, however it has a limited scope which is recommended up to 500 users and if you do not have a central IDP.
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:245]:
```suggestion
Konnect is included, however it has a limited scope and is therefore only recommended for up to 500 users and if you do not have a central IDP.
```

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 4]
```diff
@@ -240,6 +240,11 @@ For _best performance_, _stability_, _support_, and _full functionality_ we offi
 * openSUSE Leap 15.2 and 15.3
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:240]: Not  openSUSE Leap 15.4?

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/prerequisites/prerequisites.adoc (hunk 4 above)
Comment (hodyroff) [reply modules/ROOT/pages/prerequisites/prerequisites.adoc:240]: Different PR please

**Comment by mmattel** [comment]
Text needs rewording, working on it.

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 5]
```diff
@@ -240,6 +240,10 @@ For _best performance_, _stability_, _support_, and _full functionality_ we offi
...
 * Ubuntu 20.04 and 22.04
 
+== Usermanagement, Authorization and Authentication
```
Comment (EParzefall) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:243]:
```suggestion
== User Management, Authorization and Authentication
```

**Pull request merged**